niap-cc/Permissions/Tester/manifests/aosp-s1.xml

```diff
@@ -0,0 +1,6305 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Did we need these platform manifests in the permission tester project, or can we access them from cs.android.com?

Reply:
we can access them from cs.android.com. it's redundant. let me remove this directory...